packages/applicant/src/pages/mandatory-questions/[mandatoryQuestionId]/organisation-summary.page.tsx

```diff
@@ -172,16 +172,18 @@ export default function MandatoryQuestionOrganisationSummaryPage({
             </dl>
 
             <Link
-              href={routes.api.mandatoryQuestions.createSubmission(
-                mandatoryQuestionId,
-                mandatoryQuestion.schemeId.toString()
-              )}
+              href={{
+                pathname: `/api/create-submission`,
+                query: {
+                  schemeId: mandatoryQuestion.schemeId.toString(),
+                  mandatoryQuestionId,
+                },
+              }}
             >
               <a
                 className="govuk-button"
                 data-module="govuk-button"
                 aria-disabled="false"
-                type="button"
```

Comment:
Can re-add this - might be needed for accessibility stuff

Reply:
you're thinking of `role` - `type` is used to indicate the MIME type of the URL, so it clearly isn't being used correctly here and this won't do anything.